client-ts/signalr/spec/HttpConnection.spec.ts

```diff
@@ -269,6 +269,29 @@ describe("HttpConnection", () => {
         }
     });
 
+    it("does not select web socket transport when not available in environment", async (done) => {
```

Comment:
I believe "WebSocket" (one word) is the term in general use, and is used in the WebSocket Protocol spec.

https://tools.ietf.org/html/rfc6455